Add Sneh Desai

San97 opened the pull request:
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [ ] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

oisinq commented on _data/San97/San87.md:
```diff
@@ -0,0 +1,5 @@
+name: Sneh Nitin Desai
+institution:Sardar Vallabhbhai Patel Institute of Technology, Vasad
+profile_pic: sneh.jpg
+quote: Success comes to those who not only understands but also acts on creating a better tomorrow. Change as it is the only constant.
```
There's a 100 character limit on quotes, so you'll need to shorten this a bit (sorry!)

oisinq commented on _data/San97/San87.md:
```diff
@@ -0,0 +1,5 @@
+name: Sneh Nitin Desai
+institution:Sardar Vallabhbhai Patel Institute of Technology, Vasad
+profile_pic: sneh.jpg
```
The image must be at least 544x544 pixels, so this is a bit too small.

San97 commented:
Changes made as per your guidelines.

On Sun, May 24, 2020, 10:38 PM Oisín Quinn <notifications@github.com> wrote:

> *@oisinq* requested changes on this pull request.
> ------------------------------
>
> In _data/San97/San87.md
> <https://github.com/education/graduation/pull/3181#discussion_r429655978>:
>
> > @@ -0,0 +1,5 @@
> +name: Sneh Nitin Desai
> +institution:Sardar Vallabhbhai Patel Institute of Technology, Vasad
> +profile_pic: sneh.jpg
> +quote: Success comes to those who not only understands but also acts on creating a better tomorrow. Change as it is the only constant.
>
> There's a 100 character limit on quotes, so you'll need to shorten this a
> bit (sorry!)
> ------------------------------
>
> In _data/San97/San87.md
> <https://github.com/education/graduation/pull/3181#discussion_r429656196>:
>
> > @@ -0,0 +1,5 @@
> +name: Sneh Nitin Desai
> +institution:Sardar Vallabhbhai Patel Institute of Technology, Vasad
> +profile_pic: sneh.jpg
>
> The image must be at least 544x544 pixels, so this is a bit too small.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/education/graduation/pull/3181#pullrequestreview-417375155>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHO52LYYAPQZGMYUOJSNRM3RTFH77ANCNFSM4NI6YSQA>
> .
>

oisinq commented:
@San97 Just to clarify, try putting "---" at the start and end of the file.

Here's my file, if that's any help: https://github.com/education/graduation/pull/1454/files

San97 commented:
I've added the headers in my file.

On Mon, May 25, 2020, 8:13 PM Oisín Quinn <notifications@github.com> wrote:

> @San97 <https://github.com/San97> Just to clarify, try putting "---" at
> the start and end of the file.
>
> Here's my file, if that's any help:
> https://github.com/education/graduation/pull/1454/files
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/education/graduation/pull/3181#issuecomment-633604709>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHO52LZGSPUQX2FSHKEA45DRTJ7ZXANCNFSM4NI6YSQA>
> .
>

San97 commented:
Just wanted to ask.. I added the necessities in my old previous file.. so
you'll review that file right?

On Mon, May 25, 2020, 9:13 PM Sneh Desai <snehdesai97@gmail.com> wrote:

> I've added the headers in my file.
>
> On Mon, May 25, 2020, 8:13 PM Oisín Quinn <notifications@github.com>
> wrote:
>
>> @San97 <https://github.com/San97> Just to clarify, try putting "---" at
>> the start and end of the file.
>>
>> Here's my file, if that's any help:
>> https://github.com/education/graduation/pull/1454/files
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/education/graduation/pull/3181#issuecomment-633604709>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AHO52LZGSPUQX2FSHKEA45DRTJ7ZXANCNFSM4NI6YSQA>
>> .
>>
>

juanpflores commented:
I need you to add both files in the new PR so I can review them
